hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -4955,6 +4955,19 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.journalnode.edit-cache-size.fraction</name>
+  <value>0.5f</value>
+  <description>
+    This ratio refers to the proportion of the maximum memory of the JVM.
+    Used to calculate the size of the edits cache that is kept in the JournalNode's memory.
+    The recommended value is less than 0.9. We recommend using
+    dfs.journalnode.edit-cache-size.fraction instead of
+    dfs.journalnode.edit-cache-size.bytes. If we set
+    dfs.journalnode.edit-cache-size.bytes, this parameter will not take effect.
```

Comment:
I don't think we specifically recommend one or the other? They are just different ways to configure it.